HotelBooking/Views/book_reservation_view.py

```diff
@@ -44,6 +45,11 @@
         'message': "Enter the number of days you want to reserve",
         'name': 'duration',
     }],
+    "accomodation": [{
```

Comment:
Sorry I missed this last time and in your previous PR.

For Y/N use PyInquiry `'type': 'confirm'` which returns a boolean. 

and for RegistrationView, can you change the password to password type?

https://github.com/mclhtay/CS4471/blob/main/HotelBooking/Views/registration_view.py#L30

An example can be found here: https://github.com/mclhtay/CS4471/blob/main/HotelBooking/Views/authentication_view.py#L30

Reply:
okay sure, database doesn't accept boolean though so do you want me to make the boolean a 1 or 0 to store in the DB?

Reply:
@mclhtay

Reply:
Yes. By using `type: confirm` it looks better on the UI, and it will auto-process capital/lower Yes/No. Can always map True => 1, False => 0 in where your other logic resides.

Reply:
Actually, you don't need to do post-processing of the boolean. You can use the `filter` parameter of PyInquirier to process right after. 

Try something like:

```
 'type': 'input',
 'message': "Do you require accessibility accommodations? Y/N",
'name': 'accommodation',
'filter': lambda val: 1 if val else 0,
```

Reply:
makes sense, tyty!!

Reply:
But either way works.